src/docker-in-docker/README.md

```diff
@@ -17,7 +17,7 @@ Create child containers *inside* a container, independent from the host's docker
 |-----|-----|-----|-----|
 | version | Select or enter a Docker/Moby Engine version. (Availability can vary by OS version.) | string | latest |
 | moby | Install OSS Moby build instead of Docker CE | boolean | true |
-| dockerDashComposeVersion | Default version of Docker Compose (v1 or v2) | string | v1 |
+| dockerDashComposeVersion | Default version of Docker Compose (v1 or v2 or none) | string | v1 |
```

Comment:
README files are autogenerated with https://github.com/devcontainers/action.

Can we revert the change to this file, and update 👇  instead? https://github.com/devcontainers/features/blob/89ac66a01556a2133da4cfd6a0608e80c0e42d2c/src/docker-in-docker/devcontainer-feature.json#L31